### PR TITLE
EPMRPP-45416: [GOOGLE ANALYTICS] Add new action. Collect information when launch starts by agents. Should be taken information about Client Name/Version

### PR DESCRIPTION
**Pull request by KatsiarynaTatarynovich**

### CODE REVIEW

File: lib/report-portal-client.js [hunk 1]
```diff
@@ -39,6 +41,8 @@ class RPClient {
             baseURL: this.baseURL,
             headers: this.headers,
         });
+        this.analytics = new Analytics();
+        this.analytics.setPersistentParams(params.token);
```
Comment (BorisOsipov) [inline lib/report-portal-client.js]: Are you sure you want to share user authorization token with third party service?
maybe hash it before by md5\sha-256  at least?

Comment (KatsiarynaTatarynovich) [reply lib/report-portal-client.js]: This field is superfluous in general. I removed it.

Comment (BorisOsipov) [reply lib/report-portal-client.js]: 👍 
Don't forget remove it from method call :)

Comment (KatsiarynaTatarynovich) [reply lib/report-portal-client.js]: My omission, thank you :)

**Comment by DiscoElevator** [comment]
Should we have an option to disable analytics? At least, it would be useful for local development.

UPD: after discussion, decided that it shouldn't be controlled, at least in the config.

**Review by DiscoElevator — APPROVED** [review]

**Pull request merged**